lib/controller.js

```diff
@@ -747,18 +747,28 @@ controller.update = function(opts) {
 };
 
 controller.updateTesselWithVersion = function(force, tessel, currentVersion, build) {
+  function logCompletion() {
+    if (!force) {
+      logs.info('Updated', tessel.name, 'from ', currentVersion, ' to ', build.version);
+    } else {
+      logs.info('Force updated', tessel.name, 'to version', build.version);
+    }
+  }
 
```

Comment:
@rwaldron that's what @aricearice is doing here. Are you suggested she explicitly assign `tessel.name` to a new variable prior to initiating restart in case `tessel.name` gets set automatically somehow in the future?

The real hard part of this PR is that the Tessel continues to boot for another 30 seconds or so after the update and there is currently no easy way to figure out when it finished (so you can set the name back).